src/App.tsx

```diff
@@ -1,36 +1,34 @@
-import styled, { ThemeProvider } from 'styled-components'
+import { useMemo } from 'react'
+import { ThemeProvider } from 'styled-components'
 import 'antd/dist/antd.css'
 import Routers from './routes'
 import Toast from './components/Toast'
-import withProviders from './contexts/providers'
+import withProviders, { useAppState } from './contexts/providers'
 import useInitApp from './contexts/providers/hook'
-import {
-  MAINNET_PRIMARY_THEME_COLOR,
-  MAINNET_SECONDARY_THEME_COLOR,
-  TESTNET_PRIMARY_THEME_COLOR,
-  TESTNET_SECONDARY_THEME_COLOR,
-} from './constants/common'
 import { isMainnet } from './utils/chain'
 
-const AppDiv = styled.div`
-  width: 100vw;
-  height: 100vh;
-`
-
-const Theme = {
-  primary: isMainnet() ? MAINNET_PRIMARY_THEME_COLOR : TESTNET_PRIMARY_THEME_COLOR,
-  secondary: isMainnet() ? MAINNET_SECONDARY_THEME_COLOR : TESTNET_SECONDARY_THEME_COLOR,
+const appStyle = {
+  width: '100vw',
```

Comment:
Use `100%` instead of `100vw` or add `max-width: 100%`
Ref: https://stackoverflow.com/a/23367521/5579894

Reply:
I try add two box with `{width: 100vw; height: 100vh}`, there is no horizontal scroll bar.